packages/myst-parser/tests/roles/si.yml

```diff
@@ -32,3 +32,22 @@ cases:
                   unit: Hz
                   units: [hertz]
                   value: 100 Hz
+
+
+  - title: decimal number parses
```

Comment:
Can you add a couple more test cases (hopefully easy copy/paste)? At least one with a negative, maybe one with only a decimal, no exponent?

Hmm, I think `.345 <\hertz>` might fail still... That's totally fine with me, just another case to consider (maybe add a failing case for this one with `error: true` if you agree that failing is the right behaviour). Could also add a failing case for point-delimited numbers, eg. `1.000.000,05` 🤷‍♀️